src/js/components/container/SettingsButton.jsx

```diff
@@ -24,20 +27,26 @@ class SettingsButton extends Component {
     })
   }
 
-  renderSetting (setting) {
-    // Currently all settings are boolean. In order to support generic settings, this option needs to be expanded
+  renderSetting(setting) {
+    var settingControl = null;
+    switch(typeof(setting.value)) {
+      case 'boolean':
+        settingControl = <Switch setting={setting} onUpdate={() => Settings.set(setting.key, !setting.value)}></Switch>
+        break;
+      
+      case 'number':

```

Comment:
Maybe change this to number_slider or something like that, In the future maybe we will want a number input.
@idanstark42 what do you think?

Reply:
Maybe... But I'm not sure. UX speaking you should use consistent inputs, but on the rate Cade that we'll need a numérica input as well, we'll change the name. I don't think it's needed here yet